lib/conditionals.md

```diff
@@ -5,8 +5,8 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is a conditional?
 
-Replace me with your response...
+A conditional is a piece of code that will only execute if a certain conditional statement is true.
```

Comment:
Nice. One terminology suggestion. In:

> A conditional is a piece of code...

Instead of "piece", I'd suggest "block".